src/pages/index.tsx

```diff
@@ -1,83 +1,47 @@
+import Header from "@pb/components/header";
 import { type NextPage } from "next";
-import Head from "next/head";
-import Link from "next/link";
-import { signIn, signOut, useSession } from "next-auth/react";
+import { type CardData } from "@pb/types";
 
-import { api } from "~/utils/api";
+const cardData: CardData[] = [
+  {
+    title: "Daily Standup",
+    description: "Track daily updates and blockers on autopilot",
+    flowID: "daily-standup",
+  },
+  {
+    title: "Quick Check-in",
+    description: "Keep teammates in the loop with a three-question survey",
+    flowID: "quick-check-in",
+  },
+  {
+    title: "Blockers Report",
+    description: "Give your team an easy way to flag the blockers",
+    flowID: "blockers-report",
+  },
+  // Add more card data here
+];
 
 const Home: NextPage = () => {
-  const hello = api.example.hello.useQuery({ text: "from tRPC" });
-
   return (
     <>
-      <Head>
-        <title>Create T3 App</title>
-        <meta name="description" content="Generated by create-t3-app" />
-        <link rel="icon" href="/favicon.ico" />
-      </Head>
-      <main className="flex min-h-screen flex-col items-center justify-center bg-gradient-to-b from-[#2e026d] to-[#15162c]">
-        <div className="container flex flex-col items-center justify-center gap-12 px-4 py-16 ">
-          <h1 className="text-5xl font-extrabold tracking-tight text-white sm:text-[5rem]">
-            Create <span className="text-[hsl(280,100%,70%)]">T3</span> App
-          </h1>
-          <div className="grid grid-cols-1 gap-4 sm:grid-cols-2 md:gap-8">
-            <Link
-              className="flex max-w-xs flex-col gap-4 rounded-xl bg-white/10 p-4 text-white hover:bg-white/20"
-              href="https://create.t3.gg/en/usage/first-steps"
-              target="_blank"
-            >
-              <h3 className="text-2xl font-bold">First Steps →</h3>
-              <div className="text-lg">
-                Just the basics - Everything you need to know to set up your
-                database and authentication.
-              </div>
-            </Link>
-            <Link
-              className="flex max-w-xs flex-col gap-4 rounded-xl bg-white/10 p-4 text-white hover:bg-white/20"
-              href="https://create.t3.gg/en/introduction"
-              target="_blank"
-            >
-              <h3 className="text-2xl font-bold">Documentation →</h3>
-              <div className="text-lg">
-                Learn more about Create T3 App, the libraries it uses, and how
-                to deploy it.
-              </div>
-            </Link>
-          </div>
-          <div className="flex flex-col items-center gap-2">
-            <p className="text-2xl text-white">
-              {hello.data ? hello.data.greeting : "Loading tRPC query..."}
-            </p>
-            <AuthShowcase />
-          </div>
+      <HeadMeta title="PepperBot | Demo" />
+      <main className="min-h-screen bg-base-100">
```

Comment:
What are your thoughts on grouping the classes "**min-h-screen bg-base-100**" together in a class defined within the **globals.css** file? By doing this, we can conveniently apply the same styling to all future pages we create.

Reply:
good suggestion we can use it from now